modules/distr-tracing-tempo-config-spanmetrics.adoc

```diff
@@ -96,11 +96,13 @@ spec:
         monitorTab:
           enabled: true # <1>
           prometheusEndpoint: https://thanos-querier.openshift-monitoring.svc.cluster.local:9091 # <2>
+          redMetricsNamespace: "" <3>
         ingress:
           type: route
 ----
 <1> Enables the monitoring tab in the Jaeger console.
 <2> The service name for Thanos Querier from user-workload monitoring.
+<3> Optional: The metrics namespace on which the Jaeger query retrieves the Prometheus metrics. If you omit this line, the `traces.span.metrics` namespace will be used. Alternatively, you can specify the namespace manually or as `""` to use the same namespace as used by the OpenTelemetry Collector.
```

Comment:
@rubenvp8510, can please check this to see if this is correct? If not, please provide a suggestion to correct it.

Reply:
I think we should specify here that empty namespace is only if you want to make it work for OpenTelemetry Collector versions prior to 0.109.0